src/api/style/style.api.utils.ts

```diff
@@ -534,12 +553,20 @@ export async function pollWorkflowForResult<T>(
   workflowId: string,
   config: Config,
   styleOperation: StyleOperationType,
+  timeout: number = 300000,
+  startTime: number = Date.now(),
 ): Promise<T> {
   let attempts = 0;
   const maxAttempts = 30;
```

Comment:
The max attempts need to be adjusted as per time in seconds. Example in 300 seconds you would have timeout in seconds / poll Interval in seconds. Example 300/2

Reply:
Updated the attempt count to be `Math.floor(imeout_in_seconds / poll_Interval_in_seconds)`, likely it will there for be never used now.